### ISSUE TITLE
Organization proposal

### ISSUE THREAD

**Issue by danisev7** [Encode-Solidity-Bootcamp-Group-10/Week1#4]
To make the pull requests for this project, I propose to do a lightweight feature branches.

It will be the same feature branches strategy proposed at [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow
) but without the use of development branch because this is a very small project.

![image](https://user-images.githubusercontent.com/1956176/178116597-143fcbab-75b2-40fe-bfa7-b5b07f7f5964.png)

Basically the workflow will be:

- To find a peace of code to improve
- To open a new branch with the name feature-xxxxx
- To do a pull request for this branch
